pybossa/view/projects.py

```diff
@@ -3845,7 +3845,7 @@ def assign_users(short_name):
     ensure_authorized_to('read', project)
     ensure_authorized_to('update', project)
     access_levels = project.info.get('data_access', None)
-    if not data_access_levels or not access_levels:
+    if data_access_levels and not access_levels:
```

Comment:
projects cannot be configured w/o setting access levels. hence, it seems original condition can remain unchanged.

Reply:
updated via ccc29e5